src/inscription.rs

```diff
@@ -15,24 +16,42 @@ const PROTOCOL_ID: &[u8] = b"ord";
 
 const BODY_TAG: &[u8] = &[];
 const CONTENT_TYPE_TAG: &[u8] = &[1];
+const PARENT_TAG: &[u8] = &[3];
 
 #[derive(Debug, PartialEq, Clone)]
 pub(crate) struct Inscription {
-  body: Option<Vec<u8>>,
+  parent: Option<InscriptionId>,
```

Comment:
This is a more general point, but I was curious if there is a technical reason behind having only a single parent (vs. a list of 1 or more)?

I understand that in most cases people may want just one but there are also lots of usecases where you may want more than one. A basic example is a mashup (I take one inscription and another inscription and create a new mashup art). A more advanced example is "crafting", where I may take some set of inscriptions as parents, create a child, and burn the parents (all in one tx).